go.mod

```diff
@@ -1,4 +1,4 @@
-go 1.23
+go 1.23.0
```

Comment:
can we use the latest version here?

Reply:
Fixed

Reply:
It looks like GitHub Actions doesn't take 1.23.5. I can update to 1.23.4.